timemachine/cpp/src/hilbert.h

```diff
@@ -32,14 +32,14 @@ typedef uint32_t halfmask_t;
  * Inputs:
  *  nDims:      Number of coordinate axes.
  *  nBits:      Number of bits per axis.
- *  index:      The index, contains nDims*nBits bits (so nDims*nBits must be <= 8*sizeof(bitmask_t)).
- * Outputs:
- *  coord:      The list of nDims coordinates, each with nBits bits.
- * Assumptions:
- *      nDims*nBits <= (sizeof index) * (bits_per_byte)
+ *  index:      The index, contains nDims*nBits bits (so nDims*nBits must be <=
+ * 8*sizeof(bitmask_t)). Outputs: coord:      The list of nDims coordinates,
```

Comment:
Breaking up comments like this isn't great, is there a way to prevent that?

Reply:
good catch, I hadn't noticed these issues. `hilbert.*` are no longer being formatted, but I'll look for other cases and see if we can disable comment formatting

Reply:
I didn't see any other cases of this, and also don't see any obvious way to disable formatting in comments. If this comes up in the future, we could probably disable formatting locally with a pragma if there's no other workaround.

Reply:
Looking at the clang-format syntax here: https://clang.llvm.org/docs/ClangFormatStyleOptions.html

I think we want the additional flag:
```
ColumnLimit: 120
```

Reply:
good call, added to e98f119

Reply:
Just a note that we might want to revisit `ColumnLimit`. Setting to `120` seems to result in some pretty long argument list lines, ex. https://github.com/proteneer/timemachine/pull/534/files#diff-87cdbf91d94f4b5f6791e9b057b3934bd68f1737aa57b7134c02a61f83c89bc2R25